kbase-extension/kbase_templates/notebook.html

```diff
@@ -16,15 +16,13 @@
 <link rel="stylesheet" href="{{ static_url("components/bootstrap-tour/build/css/bootstrap-tour.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("components/codemirror/lib/codemirror.css") }}">
 {{super()}}
-<link rel="stylesheet" href="{{ static_url("kbase/css/contigBrowserStyles.css") }}" type="text/css" />
```

Comment:
moved this to be with the other kbase stylesheets in `page.html`

Reply:
Good idea for now. I think that should be imported by the contig browser widget as requested (at least I think that's the only place it gets used). But that's a separate piece of work.